Voice/Recording Audio Tracks with Websockets/readme.md

```diff
@@ -0,0 +1,126 @@
+# Overview
+This guide will use <stream> and websockets to recieve base64 audio, which can be written to an audio file for storage, playback, or further manipulation such as transcription services. This guide will focus on taking inbound and outbound audio tracks from a call and saving them to a Wave file.
+

```

Comment:
could you add a "📖 Read the full guide" link to the guide in the developer portal? (if it exists) See for example this one: https://github.com/signalwire/guides/blob/main/Video/Simple%20Video%20Demo/README.md